src/overlays/actors/ovl_Bg_Gjyo_Bridge/z_bg_gjyo_bridge.c

```diff
@@ -21,17 +29,94 @@ const ActorInit Bg_Gjyo_Bridge_InitVars = {
     (ActorFunc)BgGjyoBridge_Update,
     (ActorFunc)BgGjyoBridge_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gjyo_Bridge/BgGjyoBridge_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gjyo_Bridge/BgGjyoBridge_Destroy.s")
+InitChainEntry initChain[] = { ICHAIN_F32(unk_F8, 800, ICHAIN_CONTINUE),
+                               ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP) };
+
+extern u32 D_06000600;
```

Comment:
id make these segment addresses UNK_PTR's until we know what type these are

Reply:
Since this is the type of whatever is at the segment address it should be UNK_TYPE until we know more.

Reply:
aren't segment addresses always pointers though? mzx was saying that on a recent pr review

Reply:
Elements in segments are referenced as pointers yeah, but this declaration corresponds to the type of whatever D_06000600 is, not the type of the pointer to D_06000600.